modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/UserModelListener.java

```diff
@@ -29,6 +31,26 @@
 @Component(service = ModelListener.class)
 public class UserModelListener extends BaseModelListener<User> {
 
+	@Override
+	public void onAfterRemove(User user) throws ModelListenerException {
+		try {
+			User defaultServiceAccountUser =
+				_userLocalService.fetchUserByScreenName(
+					user.getCompanyId(), "default-service-account");
+
```

Comment:
Hi @rijema, I believe you kept this check here to avoid errors when deleting virtual instances, but this check has been questioned before. Can you please add a comment explaining why this needs to exist here?